src/linear_algebra/basic.lean

```diff
@@ -156,6 +169,12 @@ instance linear_map_apply_is_add_monoid_hom (a : M) :
 { map_add := λ f g, linear_map.add_apply f g a,
   map_zero := rfl }
 
+@[simp] lemma add_comp (g : M₂ →ₗ[R] M₃) (h : M₂ →ₗ[R] M₃) :
+  (h + g).comp f = h.comp f + g.comp f := rfl
+
+@[simp] lemma comp_add (g : M →ₗ[R] M₂) (h : M₂ →ₗ[R] M₃) :
+  h.comp (f + g) = h.comp f + h.comp g := by { ext, simp }
```

Comment:
I'm not sure the RHS is really simpler.

Reply:
Yeah, it's not clear which way it should be. I have removed the simp tags for all of them except `comp_neg`.